.clang-format

```diff
@@ -0,0 +1,42 @@
+# Documentation: https://clang.llvm.org/docs/ClangFormatStyleOptions.html
+# Formatting
+ColumnLimit: 100
+IndentWidth: 4
+NamespaceIndentation: All
+TabWidth: 4
+UseTab: Never
+
+MaxEmptyLinesToKeep: 1
+
+# Alignment checks
+AlignOperands: AlignAfterOperator
+AlignTrailingComments: true
+AlignAfterOpenBracket: Align
+
+# Allowed shortenings
+AllowShortBlocksOnASingleLine: true
+AllowShortFunctionsOnASingleLine: All
+AllowShortIfStatementsOnASingleLine: Always
+AllowShortLoopsOnASingleLine: true
+BreakBeforeBraces: Attach
```

Comment:
Is this Allman style or 1TBS?

Reply:
This is 1TBS.

Reply:
Can you switch to Allman?

Reply:
or not enforce curly style?

Reply:
Now using clang-format's built-in Allman style.